test/prop_opt_set.erl

```diff
@@ -1,279 +0,0 @@
-
-limited_ast() ->
-  ?SIZED(Size, limited_ast(Size)).
-
-%%limited_basic_ast() ->
-%%  ?SIZED(Size, limited_basic_ast(Size)).
-
-%%limited_basic_ast(Size) when Size =< 1 ->
-%%  frequency([
-%%    {1, stdtypes:tatom()},
-%%    {10, stdtypes:tatom(common_atoms())}
-%%  ]);
-%%limited_basic_ast(Size) ->
-%%  frequency([
-%%    {10, unbalanced_binary_op(Size, fun cut_limited_basic_ast/1, [union, intersection], 6)},
-%%    {5, {negation, limited_basic_ast(Size-1)}}
-%%  ]).
-
-
-limited_ast(Size) when Size =< 1 ->
-  frequency([
-    {1, stdtypes:tatom()},
-    {10, stdtypes:tatom(common_atoms())},
-    {2, {predef, any}},
-    {1, {predef, none}},
-    {1, {tuple_any}},
-    {1, {predef, integer}},
-    {1, {fun_simple}}
-%%    {3, east:any_interval()},
-%%    {1, east:any_arrow()},
-  ]);
-limited_ast(Size) ->
-  frequency([
-%%    {10, unbalanced(Size, function)},
-    {20, unbalanced_binary_op(Size, fun cut_limited_ast/1, [union, intersection, tuple], 2)},
-    {10, unbalanced_binary_op(Size, fun cut_limited_ast/1, [union, intersection, tuple], 6)},
-    {10, {negation, (limited_ast(Size-1))}}
-]).
-
-cut_limited_ast(Size) ->
-  ?LET(CutSize, range(2, Size), limited_ast(Size div CutSize)).
-
-%%cut_limited_basic_ast(Size) ->
-%%  ?LET(CutSize, range(2, Size), limited_basic_ast(Size div CutSize)).
-
-unbalanced_binary_op(Size, Elements, Types, LimitRank) -> ?LAZY(
-  begin
-    RandomOp = oneof(Types),
-    MaxRank = LimitRank,
-    Options = [
-        {((MaxRank * 10) div Rank),
-          {RandomOp, [ Elements(Size) || _ <- lists:seq(1, Rank)]}
-        }
-      || Rank <- lists:seq(1,MaxRank)
-    ],
-    frequency(Options ++ [{1, {RandomOp, []}}]) % empty union/intersect edge case
-  end
-).
-
-%%any_int() -> oneof(['*', integer()]).
-%%non_trivial_basic(B) ->
-%%  % not empty and not ANY_BASIC
-%%  (not subtype(B, east:empty()))
-%%    andalso
-%%    (not subtype(east:any_basic(), B)).
-%%
-%%
-trivial_negation() ->
-  T = (tnegate(stdtypes:empty())),
-  T2 = (tnegate(stdtypes:any())),
-  equiv(T, stdtypes:any()) andalso equiv(T2, stdtypes:empty()).
-
-any_intersect(Ty) ->
-  T2 = (tintersect([stdtypes:any(), Ty])),
-  T3 = (tintersect([Ty, stdtypes:any()])),
-  equiv(T2, Ty) andalso equiv(T3, Ty).
-
-empty_intersect(Ty) ->
-  T2 = (tintersect([stdtypes:empty(), Ty])),
-  T3 = (tintersect([Ty, stdtypes:empty()])),
-  equiv(T2, stdtypes:empty()) andalso equiv(T3, stdtypes:empty()).
-
-any_union(Ty) ->
-  T2 = (tunion([stdtypes:any(), Ty])),
-  T3 = (tunion([Ty, stdtypes:any()])),
-  equiv(T2, stdtypes:any()) andalso equiv(T3, stdtypes:any()).
-
-empty_union(Ty) ->
-  T2 = (tunion([stdtypes:empty(), Ty])),
-  T3 = (tunion([Ty, stdtypes:empty()])),
-  equiv(T2, Ty) andalso equiv(T3, Ty).
-
-
-equiv(S, T) ->
-  subtype(S, T) andalso
-    subtype(T, S).
-
-%%
-%%equiv2(S, T) ->
-%%  subtype_old(S, T) andalso
-%%    subtype_old(T, S).
-%%
-empty(S) ->
-  equiv(S, {predef, none}).
-
```

Comment:
why did you delete these tests?

Reply:
Property-based tests will be ported separately.